tests/helpers/main.js

```diff
@@ -31,6 +32,11 @@ const zipFixture = async function (t, fixture, { length, fixtureDir, opts = {} }
   const { path: tmpDir } = await getTmpDir({
     prefix: `zip-it-test-bundler-${bundlerString}`,
   })
+
+  if (env.ZISI_KEEP_TEMP_DIRS !== undefined) {
```

Comment:
When the `ZISI_KEEP_TEMP_DIRS` variable is set, we likely want to inspect the temp directories after the tests, so it makes sense to print their path too.